spec/tests/urbanopt/geojson/zoning_spec.rb

```diff
@@ -65,14 +67,18 @@
     expect(first_floor_points[0].class).to eq(OpenStudio::Point3d)
   end
 
-  # it 'gets first floor prints' do
-  #   # NOTE: MAKE THIS MORE SPECIFIC.
-  #   path = File.join(File.dirname(__FILE__), '..', '..', '..', 'files', 'nrel_stm_footprints.geojson')
-  #   feature_id = 'Energy Systems Integration Facility'
-  #   feature = URBANopt::GeoJSON::GeoFile.new(path).get_feature_by_id(feature_id)
-  #   feature_2 = URBANopt::GeoJSON::Zoning.handle_surrounding_buildings(@runner, @origin_lat_lon, feature)
-  #   expect(feature_2[0][]).to eq(feature.feature_json)
-  # end
+  it 'gets other building spaces' do
+    path = File.join(File.dirname(__FILE__), '..', '..', '..', 'files', 'nrel_stm_footprints.geojson')
+    feature_id = '59a9ce2b42f7d007c059d2ee'
+    all_features = URBANopt::GeoJSON::GeoFile.from_file(path)
+    feature = all_features.get_feature_by_id(feature_id)
+    other_buildings = feature.create_other_buildings('ShadingOnly', all_features.json, @model, @origin_lat_lon, @runner, true)
+    expect(other_buildings[0].class).to eq OpenStudio::Model::Space
+    expect(other_buildings.size).to eq 17
+
+    # feature_2 = URBANopt::GeoJSON::Zoning.handle_surrounding_buildings(@runner, @origin_lat_lon, feature)
+    # expect(feature_2[0][]).to eq(feature.feature_json)
+  end
 
   # it 'creates a zoning floorprint from polygon' do
   # # REVISIT: WHY ZONING SET TO TRUE
```

Comment:
I still did not try to uncomment this and run.

Reply:
I uncommented it and ran, the test is passing.